### PR TITLE
chore: Add changelogs for `dunfell-v2024.06`

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

**Comment by lluiscampos** [comment]
Tag to be pushed:
```
$ git show dunfell-v2024.06 
tag dunfell-v2024.06
Tagger: Lluis Campos <lluis.campos@northern.tech>
Date:   Fri Jun 14 15:58:39 2024 +0200

June 2024 release of meta-mender/dunfell
-----BEGIN PGP SIGNATURE-----

iQJPBAABCgA5FiEEPOQUCYOEcnWaUptYlV8qwjfRfu8FAmZsTKIbHGxsdWlzLmNh
bXBvc0Bub3J0aGVybi50ZWNoAAoJEJVfKsI30X7vONcP/jyufJeYBBzcyhBHv9TL
ISoy7aB7+NVGVy48bSYYMMyV04SmKN64LfcOLAbEVtLTkvv4ij+WJykD/GUEMdFk
+9nh7c8Hb4ArtFa9neqb1U/d0NMGHhfsz03YiYgilzzjZQ5PRj6ySZ2cLCRkwm8E
ifL1PDclJ3w7K8oLyogiSjqKaMylAD1xWWrKCQPggybHsfn5Bj+viRZsD+ObN60R
0c+cNAhLZb7bz5MWFWy8nrbsuc4YazyporjSLV3mmMG/KWXAy1Gp9H0YSRagyFj4
SRiELtRN/UPp7KPbLfAflXa16MoZzphuWh9KvyR/Zk0G7eYjDlRKx36Dmk+H3DHg
ukFkZFEwVCYd0Up277PNR+FKJmELkBNG+1Hto6yKvbNZAi4ri2GD3tgHQPQPu04u
zWLgEp8T9li1W+UDjXRGIFB0TsMmc3fErS3r9Q6IqdPnBHHsA2trx+Ee9baS5b4f
sCa3eWnGw9eiwg7dC7Ni5IlQ8wNqlZMRMLsXh7yKDxonDvfhm2LoI6IC09rudp+X
1jcVD3aKZG2JFFUrdgp5FbgeWd7Ni/EZCMMxIN6num/0E2wfFzFpGKnO0RpXUooc
vuiBGCNt1IoBWD5FdqUHNkrZgLsXTpGMpAlpBfYy/fczo2SZeEv2fYVd7Tme7/XT
VZc2yONfikUldTYq9OA+Sgb3
=Un5W
-----END PGP SIGNATURE-----

commit d15404fbf01f4e96386a0602a1dee7635ff166ae (HEAD -> dunfell, tag: dunfell-v2024.06, mender/dunfell)
Merge: 5229d809 d56c3bb9
Author: Lluis Campos <lluis.campos@northern.tech>
Date:   Thu Jun 13 14:19:37 2024 +0200

    Merge pull request #2133 from lluiscampos/dunfell-mender-3.6.5
    
    Dunfell 3.6.5

```

File: 40.meta-mender/docs.md [hunk 1]
```diff
@@ -1217,6 +1217,67 @@ New changes in meta-mender since dunfell-v2022.09:
...
+    updates. This is an in-memory effect only, the physical checksum on
+    disk is unaffected.
+  * Mender-snapshot can generate corrupted rootfs dumps, for the same
+    reason as the previous point.
+* Fix typo in fstab options for extra partitions.
+* Correctly handle `mender-partuuid` feature for `dunfell`
+
+##### Features
+
+* Make the build succeed if gateway examples aren't present
```
Comment (lluiscampos) [inline 40.meta-mender/docs.md:1256]: @kacf This one I also think it should be a bugfix, right?

Comment (kacf) [reply 40.meta-mender/docs.md:1256]: Yes, I agree.

**Review by kacf — APPROVED** [review]
Good tag as well!

**Pull request merged**